src/shared/modules/commands/helpers/play.ts

```diff
@@ -18,11 +18,12 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+// Need to refactor
```

Comment:
what is this comment for? It's hard to tell without more detail...

Reply:
@jharris4 - sorry for the confusion. IMO the original `fetchRemoteGuide` API request has a conflict with the name with `fetchRemoteGuide` method that returns Redux action. I renamed it as `fetchRemoteGuideAsync` but not sure if it's semantic enough.
Also, the type needs to be specified in this file at some point.
This method seems related to `Guide`. So, I don't think it should be in `play.ts`. IMO either the file should be renamed like `guideApi.ts`, or consolidated into `guideResolverHelper.ts`. It helps enhance the readability.

Reply:
@OskarDamkjaer - how about `requests.ts.` or `api.ts` or `async.ts`? I'm wondering all API calls can be consolidated in one file in the future?

Reply:
@OskarDamkjaer - as discussed offline, we agreed to keep modules as small as possible. So, we just rename it as `playAndGuides.ts`.

Reply:
Settled on this being called playOrGuides.ts offline